README.md

```diff
@@ -36,6 +36,29 @@ npm install
 1. Run `npm start`
 2. Components are available at [http://localhost:3000]()
 
+### How to test
+
+BackstopJS is used for visual regression testing, to run visual regression tests use command
+
+```shell
+npm run test:backstop  
+```
+
+#### Caveats
+If you're running a Linux OS you may need to configure the `BACKSTOP_TEST_HOST` environment variable for the docker container used by BackstopJS. 
+To do this and run the BackstopJS tests in one command, you can run the following command
+
+```shell
+env BACKSTOP_TEST_HOST={your local machines ip address} npm run test:backstop 
+```
+
+NOTE: If you run into a problem where some of your visual regression tests are failing due to unrelated changes, for instance on screens you may not have touched or updated. 
+You may want to delete the `node_modules` folder from the root of the project and then re-run command
+
+```shell
+npm install
+```
+
```

Comment:
@kyle-bowden I think there is already documentation for `backstop` in the `CONTRIBUTING.md`, maybe that should be updated rather than adding into the `README`:
https://github.com/hmrc/hmrc-frontend/blob/main/CONTRIBUTING.md#visual-regression-tests